internal/xds/translator/basicauth.go

```diff
@@ -82,10 +82,7 @@ func buildHCMBasicAuthFilter(basicAuth *ir.BasicAuth) (*hcmv3.HttpFilter, error)
 			},
 		},
 	}
-	if err = basicAuthProto.ValidateAll(); err != nil {
```

Comment:
Validation in individual filer translators are not required anymore, as they'll be validated inside the `proto.ToAnyWithValidation` function before converting to `any`.